x-pack/plugins/security_solution/public/management/components/console/components/command_list.test.tsx

```diff
@@ -0,0 +1,155 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { ConsoleTestSetup, HelpSidePanelSelectorsAndActions } from '../mocks';
+import {
+  getCommandListMock,
+  getConsoleTestSetup,
+  getHelpSidePanelSelectorsAndActionsMock,
+} from '../mocks';
+import React from 'react';
+import { waitFor } from '@testing-library/react';
+
+describe('When rendering the command list (help output)', () => {
+  let render: ConsoleTestSetup['renderConsole'];
+  let renderResult: ReturnType<typeof render>;
+  let consoleSelectors: ConsoleTestSetup['selectors'];
+  let enterCommand: ConsoleTestSetup['enterCommand'];
+
+  beforeEach(() => {
+    const testSetup = getConsoleTestSetup();
+
+    render = (props = {}) => (renderResult = testSetup.renderConsole(props));
+    consoleSelectors = testSetup.selectors;
+    enterCommand = testSetup.enterCommand;
+  });
+
+  describe('and its displayed on the side panel', () => {
```

Comment:
Looks like all but the last two tests in this suite also have redundant `async`.

Reply:
yeah. should be ok though. I'll leave it

Reply:
I have removed the async. 👍